packages/server/modules/blobstorage/index.js

```diff
@@ -50,6 +50,16 @@ const ensureConditions = async () => {
   }
 }
 
+let fileLimit = null
+
+const getFileLimit = () => {
+  if (!fileLimit) {
+    //100Mb default
+    fileLimit = process.env.FILE_SIZE_LIMIT_BYTES || 1024 * 1024 * 100
```

Comment:
you do not need to predefine the fileLimit variable on line 53, it could only be set from sourcecode.
this func could be just

```javascript
const getFileLimit = () => {
  const defaultFileSizeLimit = 1024 * 1024 * 100
  return process.env.FILE_SIZE_LIMIT_BYTES || defaultFileSizeLimit
}
```